src/components/FormTodo/FormTodo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+type Props = {
+  setTodoTitle: (title: string) => void;
+  todoTitle: string
+  onAdd: (title: string) => void;
+};
+
+export const FormTodo: React.FC<Props> = ({
```

Comment:
```suggestion
export const TodoForm: React.FC<Props> = ({
```
maybe its more logical?